tools/asr_evaluator/asr_evaluator.py

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2022, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import json
+import random
+
+import git
+from omegaconf import OmegaConf
+
+from nemo.collections.asr.parts.utils.eval_utils import cal_write_wer, run_asr_inference, target_metadata_wer
+from nemo.core.config import hydra_runner
+from nemo.utils import logging
+
+
+"""
+This script serves as evaluator of ASR models
+Usage:
+  python python asr_evaluator.py \
+engine.pretrained_name="stt_en_conformer_transducer_large" \
+engine.inference_mode.mode="offline" \
+engine.test_ds.augmentor.noise.manifest_path=<manifest file for noise data> \
+.....
+
+Check out parameters in ./conf/eval.yaml
+"""
+
+
+@hydra_runner(config_path="conf", config_name="eval.yaml")
+def main(cfg):
+    report = {}
+    logging.info(f'Hydra config: {OmegaConf.to_yaml(cfg)}')
+
+    # Set and save random seed and git hash for reproducibility
+    random.seed(cfg.env.random_seed)
```

Comment:
What about numpy and pytorch random seeds?

Reply:
updated. Need to fix rng in perturb in another PR.